lib/chef/knife/azure_base.rb

```diff
@@ -167,8 +167,8 @@ def validate_params!
           end
         end
 
-        if locate_config_value(:winrm_transport) == "ssl" && locate_config_value(:thumbprint).nil? && ( locate_config_value(:winrm_ssl_verify_mode).nil? || locate_config_value(:winrm_ssl_verify_mode) == :verify_peer )
-          ui.error("The SSL transport was specified without the --thumbprint option. Specify a thumbprint, or alternatively set the --winrm-ssl-verify-mode option to 'verify_none' to skip verification.")
+        if locate_config_value(:winrm_ssl) == "ssl" && locate_config_value(:thumbprint).nil? && ( locate_config_value(:winrm_no_verify_cert).nil? || locate_config_value(:winrm_no_verify_cert) == :verify_peer )
+          ui.error("The SSL transport was specified without the --thumbprint option. Specify a thumbprint, or alternatively set the --winrm-no-verify-cert option to 'verify_none' to skip verification.")
```

Comment:
--winrm-no-verify-cert       Do not verify the SSL certificate of the target node for WinRM.
This is the description of --winrm-no-verify-cert so I guess if this flag is used then it considers that we ssl certification not to be done. According to me same should be reflected here ?

Reply:
Updated. Thanks